contracts/modules/assessment/IndividualClaims.sol

```diff
@@ -308,14 +326,13 @@ contract IndividualClaims is IIndividualClaims, MasterAwareV2 {
       emit MetadataSubmitted(claims.length - 1, ipfsMetadata);
     }
 
-
     require(msg.value >= assessmentDepositInETH, "Assessment deposit is insufficient");
     if (msg.value > assessmentDepositInETH) {
       // Refund ETH excess back to the sender
       (
         bool refunded,
         /* bytes data */
-      ) = msg.sender.call{value: msg.value - assessmentDepositInETH}("");
+      ) = owner.call{value: msg.value - assessmentDepositInETH}("");
```

Comment:
Should use the `owner` instead of `msg.sender` when emitting the `ClaimSubmitted` as well.